docs/python-examples/add-room-info-example.py

```
@@ -0,0 +1,143 @@
+# Add Room Information to Elements Example
+# This script shows how to add space/room assignments as properties to elements
+# Use this in a Python node after running space analysis
+
+import ifcopenshell
+import ifcopenshell.util.element
```

Ruff (0.12.2)

`ifcopenshell.util.element` imported but unused

Remove unused import: `ifcopenshell.util.element`

(F401)